### ISSUE TITLE
Bug on building ios release from downloaded code

### ISSUE THREAD

**Issue by Brocoleta** [FlutterFlow/flutterflow-issues#3885]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I downloaded the code from flutterflow, without mking any changes, when i to do a build for ios in release mode, it gives me error, im running:

flutter build ios --release --obfuscate --split-debug-info=build/app/outputs/symbols

### Expected Behavior

It must make the build successfully

### Steps to Reproduce

1. Download the code
2. Run flutter build ios --release --obfuscate --split-debug-info=build/app/outputs/symbols (if necessary configure the signing team before)

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITEek/Lqw4xJpbxI0Kr2d8JsgQMwGkQ5a7sviu5+FBQaI+OtP5YudePOQFBuWOqHTVdqAlWl/VsBwdLVk+GWVvY5PQyabrZhz7l9UxDjTk2gM5uVC8yBPnNtI/JWI3K60bu3pyRCIspvdFIiwUOMe+6+a3qCf9qOYwx5e6fDbOY=

### Visual documentation

<img width="1023" alt="Screenshot 2024-09-23 at 5 47 48 PM" src="https://github.com/user-attachments/assets/75570bd6-7d88-42f6-92b5-5c0adb4ec955">

### Environment

```markdown
- FlutterFlow version: v4.1.89+ released September 12, 2024
- Platform: Mac M1
- Browser name and version: Chrome Version 129.0.6668.58 (Official Build) (arm64)
- Operating system and version affected: macOs Sonoma 14.5
```

**Comment by Alezanello** [comment]
Hi there!

We’ve just released a new version (v5.0.0). Could you please check if this update resolves the issue you were experiencing?

**Comment by Brocoleta** [comment]
Now working, thank you!!
